### PR TITLE
Add using with Brunch

### PR DESCRIPTION
**Pull request by denar90**

### CODE REVIEW

**Comment by denar90** [comment]
@scott-w can you look at it?
Do we need to add some explanation how to configure `brunch` (example of config) or people can look at it in `brunch` doc?

File: en/getting_started/installing_marionette.md [hunk 1]
```diff
@@ -155,6 +155,36 @@ loaded before our `bundle.js` file so anything that depends on them will be able
...
+[Brunch][brunch] is a builder. Not a generic task runner, but a specialized tool 
+focusing on the production of a small number of deployment-ready files 
+from a large number of heterogenous development files or trees.
+
+To install Marionette using NPM and Brunch:
+
+  1. Install NPM following the advice from the [NPM blog][install-npm]
+  2. Install Brunch: `sudo npm install -g brunch`
+  3. Run `brunch new your_directory_name -s marionettejs`. Brunch will create simple skeleton 
+and install all needed dependancies.
```
Comment (scott-w) [inline en/getting_started/installing_marionette.md]: Spelling error - should be `dependencies`

**Comment by scott-w** [comment]
Thanks for this @denar90. I've added a comment.

Regarding configuring Brunch, is there any extra work that's needed or do those commands give you a configuration file that should just work? If it's the latter, then a link to the Brunch docs will suffice. If extra configuration is needed then it'll be good to see a working configuration file that the reader could copy and paste.

**Comment by denar90** [comment]
`brunch` configuration file will be present after installation. I've added only link to doc.

**Comment by scott-w** [comment]
:+1:

**Pull request merged**